package.json

```diff
@@ -38,7 +38,7 @@
         "vscode-test": "^1.4.0"
     },
     "extensionDependencies": [
-        "ms-vscode.cpptools"
+        "vadimcn.vscode-lldb"
```

Comment:
The explicit extension dependency one any extension in `package.json` is no longer required so this can go entirely. People can install the particular debugger they'd like to use.